platform-dom/src/main/java/com/softicar/platform/dom/document/marker/IDomDocumentMarkerHolder.java

```diff
@@ -20,6 +20,8 @@
 
 	/**
 	 * Defines the marker for the given {@link IDomNode}.
+	 * <p>
+	 * TODO This method should be rather called <i>addMarker</i>.
```

Comment:
We should add a reference to `PLAT-568` here.

Reply:
Also, "marker" methods should be pluralized, i.e. `setMarker -> setMarkers`, `hasMarker -> hasMarkers`, `withMarker -> withMarkers`

Reply:
ok